themes/university.typ

```diff
@@ -0,0 +1,263 @@
+// University theme
+//
+// By Pol Dellaiera - https://github.com/drupol
+//
+// Please feel free to improve this theme
+// by submitting a PR in https://github.com/andreasKroepelin/typst-slides
+//
+#let university-theme(
+    university-name: "University",
+    color-a: none,
+    color-b: none,
+    color-c: none,
+    logo: none
+) = data => {
+    let color-title = color-a
+    let color-subtitle = color-a
+    let color-section = color-a.lighten(85%)
+    let color-authors = black
+
+    let color-footer-fill-short-authors = color-a
+    let color-footer-fill-short-title = color-b
+    let color-footer-fill-date = color-b
+    let color-footer-fill-slide-counter = color-b
+
+    let color-footer-short-authors = white
+    let color-footer-short-title = white
+    let color-footer-date = white
+    let color-footer-slide-counter = white
+
+    let section = state("section", none)
+    let logical-slide = counter("logical-slide")
+    let subslide = counter("subslide")
+    let total-slides = counter("slide")
```

Comment:
Those variables are already globally present from `slides.typ`, you do not need to redefine them here again.

Reply:
If I remove them, it simply does not work.